fedn/utils/helpers/plugins/binaryhelper.py

```diff
@@ -0,0 +1,16 @@
+from fedn.utils.helpers.plugins.numpyhelper import Helper
```

Comment:
Would it improve readability to do: 

from fedn.utils.helpers.plugins.numpyhelper import Helper as NumpyHelper 
..
class Helper(NumpyHelper) 

?